spec/lib/generators/atmosphere/install_generator_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+require 'generator_spec'
+require 'generators/atmosphere/install_generator'
+
+describe Atmosphere::Generators::InstallGenerator, type: :generator do
+  destination File.expand_path("../../tmp", __FILE__)
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.